reflex/event.py

```diff
@@ -567,6 +567,23 @@ def set_clipboard(content: str) -> EventSpec:
     )
 
 
+def scroll_to(elem_id: str) -> EventSpec:
+    """Select the id of a html element for scrolling into view.
+
+    Args:
+        elem_id: the id of the element
+
+    Returns:
+        EventSpec: an event to scroll the page to the selected element
+
+    """
+    elem_id = "#" + elem_id
```

Comment:
I don't think we need the preceeding `#` character. we should use the elem_id as provided